### ISSUE TITLE
[Feature request] Integrate well-known addresses and/or address book

### ISSUE THREAD

**Issue by kopeboy** [nanopay/nanopay.me#7]
Would be nice if I could create an invoice (especially if from the UI) by typing an alias like `kopeboy`, or `kopeboy@xno.link` or `kopeboy@nano.to`, that is autocompleted with existing third-party or your own registry services.

**Current implementations** (AFAIK): [nano.to](https://nano.to/), [xno.link](https://xno.link/)

**Specification** (AFAIK): [commit file](https://github.com/mistakia/nano-community/blob/b59c11244c04081a34cd3b971d122db9e5753feb/docs/getting-started-devs/integrations.md#nano-internet-identifiers), [PR](https://github.com/mistakia/nano-community/pull/72)

**Comment by anarkrypto** [comment]
@kopeboy The idea would be to type the alias of whoever is going to pay the invoice?

**Comment by kopeboy** [comment]
No, currently the "Recipient Nano address", which we have to manually type (or paste). But in the future any address field could be autocompleted, why not.

**Comment by anarkrypto** [comment]
@kopeboy, thanks for the suggestion. I will be studying this feature and let you know when there is progress.
